gemini/tests/tree-view.js

```diff
@@ -0,0 +1,67 @@
+var WAIT_TIME = 5000;
+var WAIT_LOAD_TIME = 1000;
+
+gemini.suite('tree-view', (child) => {
+
+        gemini.suite('Tree Node', (child) => {
+            child.setUrl('/tree-view/basic-tree-node')
+            .before((actions, find) => {
+                actions.waitForElementToShow('.clr-example', WAIT_TIME);
+                actions.wait(WAIT_LOAD_TIME);
+            })
+            .setCaptureElements('.clr-example')
+            .capture('default');
+        });
+
+        gemini.suite('Tree Node - DM', (child) => {
+            child.setUrl('/tree-view/basic-tree-node-dm')
+            .before((actions, find) => {
+                actions.waitForElementToShow('.clr-example', WAIT_TIME);
+                actions.wait(WAIT_LOAD_TIME);
+            })
+            .setCaptureElements('.clr-example')
+            .capture('default');
+        });
+    
+        gemini.suite('Tree View - Basic', (child) => {
+            child.setUrl('/tree-view/basic-tree-view')
+            .before((actions, find) => {
+                actions.waitForElementToShow('.row', WAIT_TIME);
+                actions.wait(WAIT_TIME); // wait for modal to fully load
+            })
+            .setCaptureElements('.row')
+            .capture('default');
+        });
+    
+        gemini.suite('Tree View Routing', (child) => {
+            child.setUrl('/tree-view/tree-node-routing')
+            .before((actions, find) => {
+                actions.waitForElementToShow('.row', WAIT_TIME);
+                actions.wait(WAIT_TIME); //  wait for modal to fully load
+            })
+            .setCaptureElements('.row')
+            .capture('default');
+        });
+
+        gemini.suite('Tree View Dynamic', (child) => {
+            child.setUrl('/tree-view/tree-view-dynamic')
+            .before((actions, find) => {
+                actions.waitForElementToShow('.row', WAIT_TIME);
+                actions.wait(WAIT_TIME); //  wait for modal to fully load
+            })
+            .setCaptureElements('.row')
+            .capture('default');
+        });
+
+        gemini.suite('Tree View Pre Population', (child) => {
```

Comment:
What do you think about adding one more screen in the case with checkboxes, so that you capture the screen after doing a click on one of them?

You can refer to `landing-page.js` for some examples on where we do mouse-over and click to take an "after" screenshot. I think it'd make our tests more complete.